defguard.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=defguard core service
+Documentation=https://defguard.gitbook.io/defguard/
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+DynamicUser=yes
+User=defguard
+ExecReload=/bin/kill -HUP $MAINPID
+EnvironmentFile=/etc/defguard/core.env
```

Comment:
@t-aleksander I see you chose to do env based config instead of toml - still I would name it like a config (doesn't matter that it contains ENV vars). Like: /etc/defguard/core.conf OSLT.

Reply:
alright